report/assets/styles.css

```
@@ -507,6 +533,14 @@
   line-height: 20px;
 }
 
+.lh-checklist {
+  list-style: none;
```

nit: Taking another look, I think we should drop the padding entirely
```suggestion
  list-style: none;
  padding: 0;
```